tsfresh/feature_extraction/extraction.py

```diff
@@ -344,13 +351,32 @@ def _f():
         for function_name, parameter_list in fc_parameters.items():
             func = getattr(feature_calculators, function_name)
 
+            # If the function uses the index, pass is at as a pandas Series.
+            # Otherwise, convert to numpy array
+            if getattr(func, 'input', False) == 'pd.Series':
```

Comment:
We could make this more fault tolerant and use getattr(func, 'input', False).flatten() == 'pd.series', in case somebody uses small letters for the input.

Reply:
But the input is defined in the feature calculator property, not by the user, right?